src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Officer.java

```diff
@@ -36,4 +39,8 @@
 
     @Column(name = "FORENAME2")
     private String forename2;
+
+    @ManyToOne
+    @JoinColumn(name = "STAFF_GRADE_ID")
```

Comment:
I am very scared of this DB
(fetch = FetchType.LAZY)
might be an idea here?
But ask someone who knows what they are talking about